README.md

```diff
@@ -1,2 +1,64 @@
-# tw-academy-s9
-Homework repository
+# Sample Style Guide
+Technical Writing Academy - May 2021
+
+## Bold
+
+Use bold to highlight button names, field names, menus, radio buttons, or checkboxes. For example:
+* Click the **create** button to start the process.
+* Enter the IP adress in the **IP adress** field.
```

Comment:
There is a typo in both instances of address. Also, missing a capital letter.

Reply:
Thank you! I just noticed that some applications like Soucetree do not have title case in all their buttons. Would the title case usage for buttons depend on how they are written, or are they always denoted as title case in the documentation?

Reply:
You should reference them just as they appear in the UI. In this case, I based the suggestion on the original text of the sample style guide.